lib/Qpsmtpd/Plugin.pm

```diff
@@ -347,10 +347,16 @@ sub _register_standard_hooks {
     }
 }
 
+sub db_args {
+    my ( $self, %arg ) = @_;
```

Comment:
> perl already does this for us <snip> Odd number of elements in hash assignment at -e line 1.

That error message is considerably worse than, "invalid arguments in call to db_vars, cowardly refusing to answer."